docs/dev/connectors/streamfile_sink.md

```diff
@@ -733,6 +733,9 @@ Given this, when trying to restore from an old checkpoint/savepoint which assume
 by subsequent successful checkpoints, Flink will refuse to resume and it will throw an exception as it cannot locate the 
 in-progress file.
 
+<span class="label label-danger">Important Note 4</span>: Currently, the `StreamingFileSink` only supports three filesystems: 
+HDFS/S3/Local. Flink will throw an exception when using an unsupported filesystem.
```

Comment:
HDFS/S3/Local -> HDFS, S3, and Local

"Flink will throw an exception when using an unsupported filesystem **at runtime**." I would also add the "at runtime" part as this is the user-observed behaviour. This means that the job will start successfully but it will fail as soon as it tries to write some data.